### ISSUE TITLE
[Bug]: Inpaint Masked/Whole Picture - Mask is being ignored, whole picture is inpainted

### ISSUE THREAD

**Issue by ratunku** [lshqqytiger/stable-diffusion-webui-directml#283]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

When inpainting the whole picture is being redrawn instead of only the masked are and The described object does not appear in the inpainted area.

### Steps to reproduce the problem

Note: it doesn't really matter what kind of masked content is chosen (fill, original, latent noise), the results are similar.

Img2img - Inpaint tab:
Prompt:
Wolf head pendant

Picture was uploaded:
![grafik](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/145574303/ffec30c4-0bd4-476d-a23a-1f818e19313b)

settings as below:
![grafik](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/145574303/b787e2b8-13dc-488a-a88d-3e0ec02737fd)

Result is:
Inpaint Whole Picture:
![grafik](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/145574303/9f08528d-45e5-4d8b-abc6-27927e1d1ddf)

Inpaint Masked Area Only:
![grafik](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/145574303/3c9a44dd-a112-4b1e-8fb2-9f35bf777986)

### What should have happened?

Only the masked area will be inpainted, the remaining area will be changed marginally.

### Sysinfo

[sysinfo-2023-09-26-08-14.txt](https://github.com/lshqqytiger/stable-diffusion-webui-directml/files/12724908/sysinfo-2023-09-26-08-14.txt)

### Console logs

```Shell
(Automatic1111) D:\AI\A1111_dml\stable-diffusion-webui-directml>webui.bat --onnx --backend directml --medvram
venv "D:\AI\A1111_dml\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
fatal: No names found, cannot describe anything.
Python 3.10.6 | packaged by conda-forge | (main, Oct 24 2022, 16:02:16) [MSC v.1916 64 bit (AMD64)]
Version: 1.6.0
Commit hash: 7b0b721837e4e5324d64f4073b64bbd3da0755e7
Installing onnxruntime
Installing onnxruntime-directml
WARNING! Because Olive optimization does not support torch 2.0, some packages will be downgraded and it can occur version mismatches between packages. (Strongly recommend to create another virtual environment to run Olive)
Installing Olive
Launching Web UI with arguments: --onnx --backend directml --medvram
no module 'xformers'. Processing without...
No SDP backend available, likely because you are running in pytorch versions < 2.0. In fact, you are using PyTorch 1.13.1+cpu. You might want to consider upgrading.
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
==============================================================================
You are running torch 1.13.1+cpu.
The program is tested to work with torch 2.0.0.
To reinstall the desired version, run with commandline flag --reinstall-torch.
Beware that this will cause a lot of large files to be downloaded, as well as
there are reports of issues with training tab on the latest version.

Use --skip-version-check commandline argument to disable this check.
==============================================================================
2023-09-26 10:00:10,995 - ControlNet - INFO - ControlNet v1.1.410
ControlNet preprocessor location: D:\AI\A1111_dml\stable-diffusion-webui-directml\extensions\sd-webui-controlnet\annotator\downloads
2023-09-26 10:00:11,099 - ControlNet - INFO - ControlNet v1.1.410
Model PhotoMerge-inpainting [Optimized] loaded.
Applying attention optimization: InvokeAI... done.
D:\AI\A1111_dml\stable-diffusion-webui-directml\modules\ui.py:1665: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  with gr.Row().style(equal_height=False):
D:\AI\A1111_dml\stable-diffusion-webui-directml\modules\ui.py:1788: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  with gr.Row().style(equal_height=False):
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 8.5s (prepare environment: 4.0s, import torch: 2.1s, import gradio: 1.1s, setup paths: 1.3s, initialize shared: 1.1s, other imports: 0.2s, load scripts: 1.3s, create ui: 0.7s, gradio launch: 0.2s).
Loading pipeline components...:   0%|                                                            | 0/7 [00:00<?, ?it/s]2023-09-26 10:04:11.0852000 [W:onnxruntime:, session_state.cc:1162 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2023-09-26 10:04:11.0888858 [W:onnxruntime:, session_state.cc:1164 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
Loading pipeline components...:  29%|██████████████▊                                     | 2/7 [00:00<00:00, 13.59it/s]2023-09-26 10:04:11.4248573 [W:onnxruntime:, session_state.cc:1162 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2023-09-26 10:04:11.4283460 [W:onnxruntime:, session_state.cc:1164 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
2023-09-26 10:04:11.6658155 [W:onnxruntime:, session_state.cc:1162 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2023-09-26 10:04:11.6693880 [W:onnxruntime:, session_state.cc:1164 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
Loading pipeline components...:  57%|█████████████████████████████▋                      | 4/7 [00:00<00:00,  4.79it/s]2023-09-26 10:04:12.9475182 [W:onnxruntime:, session_state.cc:1162 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2023-09-26 10:04:12.9509147 [W:onnxruntime:, session_state.cc:1164 onnxruntime::VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
Loading pipeline components...:  71%|█████████████████████████████████████▏              | 5/7 [00:02<00:01,  1.37it/s]D:\AI\A1111_dml\stable-diffusion-webui-directml\venv\lib\site-packages\transformers\models\clip\feature_extraction_clip.py:28: FutureWarning: The class CLIPFeatureExtractor is deprecated and will be removed in version 5 of Transformers. Please use CLIPImageProcessor instead.
  warnings.warn(
Loading pipeline components...: 100%|████████████████████████████████████████████████████| 7/7 [00:02<00:00,  2.52it/s]
D:\AI\A1111_dml\stable-diffusion-webui-directml\venv\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_onnx_stable_diffusion_img2img.py:37: FutureWarning: The preprocess method is deprecated and will be removed in a future version. Please use VaeImageProcessor.preprocess instead
  warnings.warn(
100%|██████████████████████████████████████████████████████████████████████████████████| 43/43 [00:02<00:00, 16.52it/s]
D:\AI\A1111_dml\stable-diffusion-webui-directml\venv\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_onnx_stable_diffusion_img2img.py:37: FutureWarning: The preprocess method is deprecated and will be removed in a future version. Please use VaeImageProcessor.preprocess instead
  warnings.warn(
100%|██████████████████████████████████████████████████████████████████████████████████| 43/43 [00:02<00:00, 17.34it/s]
D:\AI\A1111_dml\stable-diffusion-webui-directml\venv\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_onnx_stable_diffusion_img2img.py:37: FutureWarning: The preprocess method is deprecated and will be removed in a future version. Please use VaeImageProcessor.preprocess instead
  warnings.warn(
100%|██████████████████████████████████████████████████████████████████████████████████| 43/43 [00:02<00:00, 15.58it/s]
D:\AI\A1111_dml\stable-diffusion-webui-directml\venv\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_onnx_stable_diffusion_img2img.py:37: FutureWarning: The preprocess method is deprecated and will be removed in a future version. Please use VaeImageProcessor.preprocess instead
  warnings.warn(
100%|██████████████████████████████████████████████████████████████████████████████████| 43/43 [00:02<00:00, 17.60it/s]
```
